tests/test_component_view_render.py

```diff
@@ -0,0 +1,12 @@
+from formiodata.form import Form
+from test_component import ComponentTestCase
+from formiodata.components import emailComponent
+
+
+class viewRenderEmailComponent(ComponentTestCase):
```

Comment:
Please rename file to `test_view_render_component`.
So distinguishing the list of test files becomes easier, regarding components and other specific tests.